installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{- define "service.image" -}}
```

Comment:
You named the helper function `service.image` but then the usage comment below talks about `common.images.image`. I think one of the 2 names is wrong here...
I would stick to `common.images.image` since this follows the chart.template-file.helper-function naming convention that we use in other common chart files as well.